plugins/modules/database/misc/kibana_plugin.py

```diff
@@ -59,6 +59,11 @@
       - Delete and re-install the plugin. Can be useful for plugins update.
       type: bool
       default: 'no'
+    allow_root:
+      description:
+      - Whether to allow C(kibana) and C(kibana-plugin) to be run as root. Passes the C(--allow_root) flag to these commands.
+      type: bool
+      default: 'no'
```

Comment:
(`false` is a boolean, the string `'no'` is not)
```suggestion
      default: false
      version_added: 2.3.0
```

Reply:
Thank you!